integration-tests/operator/amazoncloudwatchagent.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: cloudwatch.aws.amazon.com/v1alpha1
```

Comment:
we can delete this file now that we are using helm templates

Reply:
done!!